rero_ils/modules/notifications/api.py

```diff
@@ -187,12 +188,12 @@ def replace_pids_and_refs(self):
             patron = Patron.get_record_by_pid(data['loan']['patron']['pid'])
             view_code = patron.get_organisation().get('code')
             base_url = current_app.config.get('RERO_ILS_APP_URL')
-            profile_url = f'{base_url}/{view_code}/patrons/profile'
+            next_page_url = urllib.parse.quote(f'{view_code}/patrons/profile')
+            profile_url = f'{base_url}/signin/?next={next_page_url}'
             data['loan']['profile_url'] = profile_url
-
             return data
         except Exception as error:
-            raise(error)
+            raise error
```

Comment:
I think you have to write something like `raise Exception(error)`.

Reply:
`error` is already an `Exception`subclass. Using parentheses to raise an exception isn't a best practice.
Tested on localhost and works as expected.